content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -50,58 +50,97 @@ This example cron job config `.spec` file prints the current time and a hello me
 Run the example cron job by downloading the example file and then running this command:
 
 ```shell
-$ kubectl create -f ./cronjob.yaml
-cronjob "hello" created
+curl -sO https://kubernetes.io/examples/application/job/cronjob.yaml
+kubectl create -f ./cronjob.yaml
```

Comment:
instead of having curl, can you please change kubectl like below?

`kubectl create -f https://k8s.io/examples/application/job/cronjob.yaml`

Reply:
@DanyC97 Got it.